docs/guide/page/page_block_attributes.md

```diff
@@ -156,4 +156,38 @@ Validators can be also set on a parent attribute (group defining level), it mean
 
 ``` yaml
 [[= include_file('code_samples/page/custom_page_block/config/packages/nested_attribute.yaml', 9,16) =]] [[= include_file('code_samples/page/custom_page_block/config/packages/nested_attribute.yaml', 19,26) =]]
-```
\ No newline at end of file
+```
+
+## Help messages for form fields
+
+With the `help`, `help_attr` and `help_html` field options, you can set help messages for fields in the Page block.
+
+You can set options with the following configuration:
+
+```yaml
+ezplatform_page_fieldtype:
+    blocks:
+        block_name:
+            attributes:
+                attribute_name:
+                    options:
+                        help:
+                            text: 'Some exemplary text'
+                            html: true|false
+                            attr:
+                                class: 'class1 class2'
+```
+
+- `help` - defines a help message which is rendered below the field.
+- `help_attr` - sets the HTML attributes for the element which displays the help message.
+- `help_html` - set this option to `true` to not escape the contents of the `help` option from rendering them in the template.
+
+### Help message in nested attributes
+
+You can set the options for root or nested attribute, see the exemplary configuration:
+
+```yaml
+[[= include_file('code_samples/page/custom_attribute/config/packages/help_messages.yaml') =]]
+```
+
+![Help message](../img/page_block_help_message.png "Help message")
```

Comment:
```suggestion
![Help message](page_block_help_message.png "Help message")
```
This should actually work, Autolinks applies for images as well as regular pages.